website/docs/r/bot_channels_registration.markdown

```diff
@@ -43,8 +43,16 @@ The following arguments are supported:
 
 * `microsoft_app_id` - (Required) The Microsoft Application ID for the Bot Channels Registration. Changing this forces a new resource to be created.
 
+* `cmek_key_vault_url` - (Optional) The CMEK Key Vault Key URL to encrypt the Bot Channels Registration with the Customer Managed Encryption Key.
```

Comment:
given that every other resource uses cmk
```suggestion
* `cmk_key_vault_url` - (Optional) The CMEK Key Vault Key URL to encrypt the Bot Channels Registration with the Customer Managed Encryption Key.
```

Reply:
Updated